common/templates/common/home_page.html

```diff
@@ -7,12 +7,23 @@
 
 {% block content %}
         {% header %}
-        <div class='content'>{% for b in page.content %}
-                {% if b.block_type == 'heading' %}
-                        <h1 class='caps'>{{ b.value }}</h1>
-                {% else %}
-                        {% include_block b %}
-                {% endif %}
-        {% endfor %}</div>
+        <div class="page-container">
+            <!-- BEGIN CONTAINER -->
+                <div class="container min-hight margin-top-50">
+                <!-- BEGIN OUR BOARD -->
```

Comment:
I'm not sure we need all of this markup. Can we just change the class from content to container?